app/forms/concerns/hyrax/contained_in_works_behavior.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+module Hyrax
+  ##
+  # A module of form behaviours common to PCDM Objects and FileSets.
+  module ContainedInWorksBehavior
```

Comment:
is this the right name for this thing?

it seems like it's a bit of a funny group of behaviors that are incidentally common between the forms, and could be roughly broken down into: 

  - "In Works" handling
  - deposit agreement (does the FileSet form actually need this?)
  - depositor property
  - visibility/permissions/embargo|lease

i'm fine naming this as-is if we think we might move toward breaking the non "In Works" parts out to other modules later. just trying to get some clarity on what abstraction(s) we're trying to develop.